pvt/helpers/src/models/pools/weighted/types.ts

```diff
@@ -204,11 +204,8 @@ export type ManagedPoolRights = {
 };
 
 export type ManagedPoolParams = {
```

Comment:
I would consider refactoring these types so that they match the ones in the contracts.

This one e.g. is the settings params, and we are missing the new `ManagedPoolParams` and `ManagedPoolConfigParams`.

Reply:
I didn't do the refactor because I'm not a fan of how this ended up working (as mentioned in the original comment), and would rather change how this whole thing works tbh.